data/json/items/melee/bludgeons.json

```
@@ -1485,5 +1485,24 @@
     "to_hit": { "grip": "weapon", "length": "long", "surface": "line", "balance": "good" },
     "price": 16000,
     "qualities": [ [ "HAMMER", 1 ] ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "millwall_brick",
+    "name": { "str": "Millwall brick" },
+    "description": "An improvised club, made of a tightly rolled newspaper, folded lengthwise and duct-taped.  It packs a surprising punch for what is essentially paper.",
+    "weight": "70 g",
+    "price": 0,
+    "price_postapoc": 0,
+    "color": "light_gray",
+    "symbol": "/",
+    "looks_like": "cudgel",
+    "material": [ "paper", "plastic" ],
+    "techniques": [ "RAPID" ],
+    "weapon_category": [ "MACES" ],
+    "volume": "200 ml",
+    "longest_side": "30 cm",
+    "bashing": 10,
```

```suggestion
    "bashing": 3,
```
This should be more reasonable for rolled-up newspaper.